docs/getting-started-guides/ubuntu.md

```diff
@@ -246,6 +247,42 @@ the latter one could start it again.
 4. You can also customize your own settings in `/etc/default/{component_name}`.
 
 
+### Upgrading a Cluster
+
+If you already have a kubernetes cluster, and want to upgrade to a new version,
+you can use following command in cluster/ directory to update the whole cluster or a specified node to a new version.
+
+```console
+KUBERNETES_PROVIDER=ubuntu ./kube-push.sh [-m|-n <node id>] <version>
+```
+
+It can be done for all components (by default), master(`-m`) or specified node(`-n`).
+If the version is not specified, the script will try to use local binaries.
+Upgrading single node is experimental now. You can use following command to get a help.
+
+```console
+KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -h
+```
+
+Some examples are as follows:
+
+* upgrade master to version 1.0.5: `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -m 1.0.5`
+* upgrade node 10.10.103.223 to version 1.0.5 : `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -n 10.10.103.223 1.0.5`
+* upgrade master and all nodes to version 1.0.5: `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh 1.0.5`
+
+The script will not delete any data of your cluster, it just replace the binaries.
+You can use `kubectl` command to check if the newly upgraded k8s is working correctly.
+For example, use `$ kubectl get nodes` to see if all of your nodes are ready.
+
+```console
+$ kubectl get nodes
+NAME            LABELS                                 STATUS
+10.10.103.162   kubernetes.io/hostname=10.10.103.162   Ready
+10.10.103.223   kubernetes.io/hostname=10.10.103.223   Ready
+10.10.103.250   kubernetes.io/hostname=10.10.103.250   Ready
+```
```

Comment:
I strongly recommend that we should deprecate this 'cause it's only the dup of https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/ubuntu.md#test-it-out, where users could get the idea from.